src/modules/Transcript/index.tsx

```diff
@@ -320,23 +333,24 @@ export const TranscriptUI = (props: TranscriptUIProps): ReactElement => {
             {match(eventQuery)
                 .with({ status: 'success' }, { status: 'empty' }, ({ data: { events } }) => {
                     const event = events[0];
-                    const primaryQuote = getPrimaryQuote(event?.primaryCompany);
                     return (
                         (event?.audioRecordingUrl || event?.isLive) && (
                             <Playbar
-                                hidePlayer={!showPlayer}
                                 hideEventDetails={useConfigOptions}
+                                hidePlayer={!showPlayer}
                                 id={event?.id}
+                                metaData={{
+                                    createdBy: getEventCreatorName(event?.creator as User),
```

Comment:
Need to cast to `User` here since different event queries could have different definitions of `creator`, so need to standardize to `User`